app.js

```diff
@@ -19,6 +19,7 @@ app.use((req, res) => {
 });
 
 app.use((err, req, res, next) => {
+  console.log(err.message);
```

Comment:
To ma być wiadomość dla ciebie czy innych deweloperów więc może warto się zastanowić czy nie lepiej console całego err?